rules_building_block/discovery_suspicious_which_command_execution.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2023/08/30"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/08/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for the usage of the which command with an unusual amount of process arguments. Attackers may leverage
+the which command to enumerate the system for useful installed utilities that may be used after compromising a system to
+escalate privileges or move latteraly across the network. 
+"""
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Suspicious which Enumeration"
```

Comment:
```suggestion
name = "Suspicious Which Enumeration"
```

Reply:
I think commands such as "which" or "bash" should not be addressed with a capital letter right? Or at least when I Googled it, I saw that most command references are done without a capital letter, given that Unix is capital sensitive. WDYT?